### ISSUE TITLE
[Feature Request] Customizable guild features

### ISSUE THREAD

**Issue by noirscape** [powercord-org/powercord#196]
**Describe the feature request**
Having the ability to locally set a guild banner.

**What Plugin would it be for?**
New plugin.

**Expected behavior**
When selecting a guild, some guilds are boosted enough (or are paid for game servers) to show a guild banner. It would be fancy and nice to be able to set the guild banner manually, even if the guild does not have a boosted banner.

At the moment this behavior is not in Powercord but is provided by a similar plugin for BetterDiscord called [EditServers](https://raw.githubusercontent.com/mwittrien/BetterDiscordAddons/8bc7a19ec2531caab1e36aaa6d74c5d267948035/Plugins/EditServers/EditServers.plugin.js), which might help with development if this is to be implemented.

**Comment by aetheryx** [comment]
in addition to letting users set banners locally, i think it's a good idea to parse these serverside for partnered servers and/or donators similar to guild badges

**Comment by davidcralph** [comment]
Possible to also get animated server icons? (In the style of nitro boosts)

**Comment by aetheryx** [comment]
i'm fairly impartial to animated server icons. @Bowser65?

**Comment by cyyynthia** [comment]
I'm unsure about server icons. I feel like this would be bypass Nitro just to bypass Nitro and I don't know if this should be the role of a client mod. I can understand people wants features like banners but animated icons are such a small addition it can be ignored. For me the role of a client mod is to enhance Discord experience, not bypass Discord's limitations (even if doing so can improve experience)

tl;dr i'm ok with custom banners, but not totally for animated server icons

**Comment by davidcralph** [comment]
fair enough

**Comment by cyyynthia** [comment]
Moved this to powercord-community/suggestions#102 as this is a plugin request rather than a Powercord core feature
